README.md

```diff
@@ -25,7 +26,11 @@ line-bot-sdk-nodejs documentation: https://line.github.io/line-bot-sdk-nodejs/#g
 Using [npm](https://www.npmjs.com/):
 
 ``` bash
-$ npm install @line/bot-sdk --save
+$ npm install @line/bot-sdk
+
+$ yarn add @line/bot-sdk
+
+$ pnpm add @line/bot-sdk
```

Comment:
I don't see much advantage in writing this, but since the maintenance cost is nearly zero, this change adds them. For some reason(but I'm not sure), all well-known repositories have it included.